src/index.ts

```diff
@@ -1,20 +1,21 @@
+import {
+  IAssignmentLogger,
+  validation,
+  constants,
+  ExperimentConfigurationRequestor,
+  IEppoClient,
+  EppoClient,
+  HttpClient,
+  IConfigurationStore,
+} from '@eppo/js-client-sdk-common';
+import { IExperimentConfiguration } from '@eppo/js-client-sdk-common/dist/dto/experiment-configuration-dto';
 import axios from 'axios';
 
-import { IAssignmentLogger } from './assignment-logger';
-import EppoClient, { IEppoClient } from './client/eppo-client';
+// import EppoClient, { IEppoClient } from './client/eppo-client';
```

Comment:
Did you mean to leave this here but checked in?

Reply:
Ah no, this was just commented out in the process of changing imports to `js-client-sdk-common`. Definitely does not belong here. Thanks for catching this 😅